.azure-pipelines/official-build.yml

```diff
@@ -34,11 +34,14 @@ variables:
 
 stages:
 - stage: Build
-  pool: sonicbld
+  pool: sonicbld-1ES
   variables:
   - name: CACHE_MODE
     value: wcache
   - template: azure-pipelines-repd-build-variables.yml
+  preSteps:
+  - script: if which init.sh;then init.sh;fi
+    displayName: Init agent
```

Comment:
How about add a template for it, if we want to add more script, can change it easy.